src/components/current-block/KeyImages.tsx

```diff
@@ -23,7 +24,13 @@ function removeProtoBuffFromKeyImage(keyImage: string) {
     return keyImage;
 }
 
-export default function KeyImages({ blockContents }: { blockContents: Block }) {
+export default function KeyImages({
+    blockContents,
+    highlightItem
+}: {
+    blockContents: Block;
+    highlightItem: string;
```

Comment:
Optional/Equivalent: Make this optional to match the declaration in `BlockInfo`
```suggestion
    highlightItem?: string;
```